tests/restore_cluster_test.py

```diff
@@ -33,6 +33,22 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def setUp(self):
+        config = self._build_config_parser()
+        self.config = MedusaConfig(
+            storage=_namedtuple_from_dict(StorageConfig, config['storage']),
+            monitoring={},
+            cassandra=_namedtuple_from_dict(CassandraConfig, config['cassandra']),
+            ssh=None,
+            checks=None,
+            logging=None,
+            grpc=_namedtuple_from_dict(GrpcConfig, config['grpc']),
+            kubernetes=_namedtuple_from_dict(KubernetesConfig, config['kubernetes']),
+        )
+
+    @staticmethod
+    def _build_config_parser():
+        """Build and return a mutable config"""
```

Comment:
Having a mutable config allows to override it: https://github.com/thelastpickle/cassandra-medusa/pull/329/files#diff-4f9f346fe4cd28a6e2e5418122ea4c2f350d2344cd915c360c3bf127254d564aR263